cli/switch_test.go

```diff
@@ -0,0 +1,144 @@
+package cli
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type (
+	testCase struct {
+		name          string
+		args          []string
+		configExists  bool
+		configContent string
+	}
+
+	expectedOutput struct {
+		stdout string
+		stderr string
+	}
+)
```

Comment:
I know this is accepted by `gofmt` and valid Go code, tho I tend to encourage people to avoid the `type ( ... )` notation because it's not common in Go.

I'm also a big user of terminal tools, so I often grep for `type Name` to find a definition in a codebase, which this defeats. I know this one is a me-problem, but pattern searching is often useful in other contexts as well.